ruby/ruby_trunk/ruby_benchmarks/.travis.yml

```diff
@@ -0,0 +1,14 @@
+sudo: required
+
+services:
+  - docker
+
+before_install:
+  - sudo apt-get install -y curl
+  - sudo apt-get install -y jq
+
+script:
+  - sed -i "s_FROM.*_FROM rubybench/ruby_trunk:$(curl -s -S "https://registry.hub.docker.com/v2/repositories/rubybench/ruby_trunk/tags/" | jq '."results"[]["name"]'| sed -n 1p |cut -c2-8)_" ./Dockerfile
+  - docker tag $(docker build . | tail -n 1 | cut -d' ' -f 3) $rubybench/ruby_trunk:$(git ls-remote https://github.com/tgxworld/ruby trunk | cut -c1-7)
```

Comment:
Our `.travis.yml` lies in `ruby-bench-docker`, but commit sha1 belongs to the commit in `ruby`. So I found this a good way to retrieve commit sha1.